### ISSUE TITLE
Crash when reloading text file

### ISSUE THREAD

**Issue by dedyle** [TextAnalysisTool/Releases#71]
I get a crash from time to time when reloading a text file.

![image](https://user-images.githubusercontent.com/39373047/64072662-5d5ec580-cc9b-11e9-9d1d-3b329d1d9f78.png)

It seems similar to https://github.com/TextAnalysisTool/Releases/issues/53 but this time, the crash is coming from GetLineFilterNames()

**Comment by vincec-msft** [comment]
Can you give more details that would help narrow the problem? Is it consistent with a certain file? Are filters loaded? Are any lines hidden? How are you reloading the file?

**Comment by dedyle** [comment]
The file I'm reloading is being appended in the background. I'm reloading by pressing "F5".
This is not consistent. Happens maybe 1 out of 20 times I'm reloading.
Filters are loaded (some enabled some disabled) and there are hidden lines.
I started seeing this crash after I've upgraded to latest version. Before that I've used some 2017 version (maybe 2017-01-24) and the problem was not seen in this same scenario.
